models/articles.models.js

```diff
@@ -0,0 +1,20 @@
+const db = require("../db/connection");
+
+exports.fetchArticleById = (article_id) => {
+  const text = "SELECT * FROM articles WHERE article_id = $1;";
+  const params = [article_id];
+  return db.query(text, params).then(({ rows }) => {
+    if (rows.length === 0) {
+      return Promise.reject({ status: 400, msg: "Bad request" });
```

Comment:
Is it a bad request if there is no article after a valid search? In order to get to the then block in this function, psql needs to have not errored. If there was a bad request, such as api/articles/steve it would have errored before the then block. There does need to be an error here, but is it a 400?

Reply:
Changed to 404 not found